tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "module": "commonjs",
     "lib": [
-      "es6",
+      "es2021",
```

Comment:
Are the changes here (this and the ones below with `esModuleInterop`) related to the shex parser?

Reply:
Without the flag it was valid to do `import * as mkdirp from 'mkdirp'` in `lib/io/QuadSinkCsv.ts` but in my setup it was not compiling in `SolidBench` and it was needed to be `import mkdirp from 'mkdirp'`. So, I changed the flag, for the es version I just match it to be the same as SolidBench.

Reply:
The solution by @wvdemeer might be better here: https://github.com/SolidBench/rdf-dataset-fragmenter.js/compare/master...wvdemeer:rdf-dataset-fragmenter.js:mkdirp-upgrade?expand=1

Reply:
I will delete the changes then, fix the merge conflict of this branch and make a PR.

Reply:
I don't have the authorization to make this PR so either I make a new PR with the changes or someone with the authorization does it and fix the merge conflict.

Reply:
@wvdemeer Do you want to make the PR? (since it's your code)

Reply:
I've made a PR from that compare: https://github.com/SolidBench/rdf-dataset-fragmenter.js/pull/23